third_party/vulkan_layers/amber_scoop/layer_impl.cpp

```diff
@@ -3,7 +3,7 @@
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * You may obtain a copy of the Licens
```

Comment:
This should be fixed :)

Reply:
Oops :) I didn't even mean to add the second commit to the PR.